docs/circuits.ipynb

```diff
@@ -86,7 +86,7 @@
    "source": [
     "## Conceptual overview\n",
     "\n",
-    "The primary representation of quantum programs in Cirq is the `Circuit` class. A `Circuit` is a collection of `Moments`. A `Moment` is a collection of `Operations` that all act during the same abstract time slice. An `Operation` is a some effect that operates on a specific subset of Qubits, the most common type of `Operation` is a `GateOperation`.\n"
+    "The primary representation of quantum programs in Cirq is the `cirq.Circuit` class. A `cirq.Circuit` is a collection of `Moments`. A `cirq.Moment` is a collection of `Operations` that all act during the same abstract time slice. An `cirq.Operation` is a some effect that operates on a specific subset of Qubits, the most common type of `cirq.Operation` is a `cirq.GateOperation`.\n"
```

Comment:
This may need a manual pass after replacements are made to check formatting. "An `cirq.Operation`" here should be "A `cirq.Operation`" - there may be similar minor fix-ups required.

Reply:
Yeah, it seems I only changed some of them. However, I think it'll be better to wait until a clear approach is taken.